github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -421,6 +421,15 @@ presubmits:
         resources:
           requests:
             memory: "8Gi"
+        volumeMounts:
```

Comment:
To avoid duplication maybe create a preset for this?

Reply:
Better, will do like that.